stripe/http_client.py

```diff
@@ -84,9 +84,11 @@ def request(self, method, url, headers, post_data=None):
 class RequestsClient(HTTPClient):
     name = 'requests'
 
-    def request(self, method, url, headers, post_data=None):
-        kwargs = {}
+    def __init__(self, timeout=80, **kwargs):
+        super(RequestsClient, self).__init__(**kwargs)
+        self._timeout = timeout
 
+    def request(self, method, url, headers, post_data=None, **kwargs):
```

Comment:
@alexhughson Actually, just taking one more look at this: would you mind reverting this line to how it was? I'm not sure that this change is necessary.

Could you also try rebasing the changes on master? You'll need to merge a conflict in `RequestsClientTests`, but it's not too bad.